### DISCUSSION TITLE
How do I renew expired Tailscale Key?

### DISCUSSION THREAD

**Discussion by risk00** [runtipi/runtipi-appstore#4123] [Q&A] [answered]
I installed Tailscale from within Tipi and the key is now expired. According to the docs, I need to run the command:

`sudo tailscale up --force-reauth` 

How do I do this with Runtipi?

**Comment by cori** [comment, marked as answer]
You have a couple of options here. If you have terminal access to the server that's running tipi, you should be able to open a terminal there and use this command

```
[sudo] docker exec -it tailscale /bin/sh
```

and you should get a prompt that look like

```
/ # 
```

once there you should be able to run the command you need. I did not need `sudo`, but i did receive an error message with the correct command (if you've set non-default settings in your app's config you'll need to specify those as well).

If you don't have terminal access and trust the host Tailscale is running on you can disable key expiry in the Tailscale Machines view - this is available even for expired hosts.
